README.md

```diff
@@ -31,16 +32,29 @@
 </p>
 </br>
 
```

Comment:
What do you think about adding a docs badge like the one now on the semgrep-action readme? If we’d need to remove one, I’d consider removing the Twitter badge or shortening its label.

For convenience here’s the docs badge:
```
<a href="https://semgrep.dev/docs/semgrep-ci/">
      <img src="https://img.shields.io/badge/docs-semgrep.dev-purple" alt="Documentation" />
  </a
```
I think it would go on on approx. line 29.